.github/workflows/compiler-zoo.yml

```diff
@@ -75,6 +75,10 @@ jobs:
             cc: clang-16,
             distro: ubuntu-22.04,
             llvm-ppa-name: jammy
+          }, {
+            cc: clang-17,
+            distro: ubuntu-22.04,
+            llvm-ppa-name: jammy
```

Comment:
Could you please temporarily amend this workflow so it is run on pull request? Do it via separate commit so it can be dropped either by you or when merging.

We need to see that this CI workflow will be passing.

Reply:
Sure. You were happy to take my word for it / look at my fork before. I'll do that when I get home shortly. Thanks!